src/viadot/orchestration/prefect/tasks/eurostat.py

```diff
@@ -36,11 +35,11 @@ def eurostat_to_df(
             - `column_sum`: dict: {column: {'min': number, 'max': number}}
 
     Returns:
-        pd.DataFrame: None empty DataFrame.
+        pd.DataFrame: Pandas DataFrame.
     """
 
-    data_frame = Eurostat().to_df(
-        dataset_code, params, base_url, requested_columns, tests
-    )
+    data_frame = Eurostat(
+        dataset_code=dataset_code, params=params, columns=columns, tests=tests
```

Comment:
These should be params to the `to_df()` method.

Reply:
https://github.com/dyvenia/viadot/pull/926/commits/cd1e3ec7731f4caa6ff0619a23c88f48c933e996